src/frontends/onnx/frontend/include/openvino/frontend/onnx/extension/conversion.hpp

```diff
@@ -11,9 +11,10 @@
 namespace ov {
 namespace frontend {
 namespace onnx {
-class ConversionExtension : public ConversionExtensionBase {
+class ONNX_FRONTEND_API ConversionExtension : public ConversionExtensionBase {
```

Comment:
we have explicitly removed all exports from conversion extension classes here https://github.com/openvinotoolkit/openvino/pull/28834 to avoid dependency on frontend.so in case of external extensions like openvino_tokenizers (before, they depended on TF FE because of TF FE conversion extension)

Why do we need to return it back?